### ISSUE TITLE
getTasks passing a filter with the NOT(!) operator throws a ValidationError

### ISSUE THREAD

**Issue by thales-fukuda** [Doist/todoist-api-typescript#245]
## Bug description

After calling `getTasks` and passing a filter with the `!` operator, the API response does not include the `duration` property causing the SDK to throw a `ValidationError`.

I tested doing the request manually on Postman and indeed the duration field was missing.  I also tried with a brand new account and the problem persisted.

It looks like it only happens when using the `!`, but I could be wrong.

If it is the expected behavior for the API, the `duration` field on the Task entity should be optional to stop the error.

## Expected behaviour

The request should not throw an error and it should include `duration` in the response.

## Is reproducible

Yes

## To reproduce

```js
const excludePriorityOne = async () => {
	const res = await api.getTasks({ filter: `!p1` })
	console.log(res)
}

excludePriorityOne()
```

## Version information:

-   Package version: 3.0.0
-   Node: 18.18.0

**Comment by scottwittrock** [comment]
I got a similar issue when calling `getTasks` with: 
```
{
     filter: 'overdue | today'
}
```

**Comment by henningmu** [comment]
Thank you for the report @thales-fukuda and @scottwittrock. I can reproduce this and forwarded it to our API team internally. I will let you know here once I have more information 👍

**Comment by henningmu** [comment]
Unfortunately, we'll have to (temporarily) mark the duration field as optional on the SDK side. The filter API is different from the sync API (long story), and hence, there are some type inconsistencies. There are plans in motion to unify all this during next year, and by then we can start relying on the duration property. For now, it won't be returned.

@thales-fukuda are you up for another PR? 🙌

**Comment by thales-fukuda** [comment]
> Unfortunately, we'll have to (temporarily) mark the duration field as optional on the SDK side. The filter API is different from the sync API (long story), and hence, there are some type inconsistencies. There are plans in motion to unify all this during next year, and by then we can start relying on the duration property. For now, it won't be returned.

Got it! Great to hear that are plans to unify the APIs. Thanks for the update!

> @thales-fukuda are you up for another PR? 🙌

Yes, absolutely!
